client/src/PublicRequest.ts

```diff
@@ -161,7 +161,8 @@ export type IFrameResult = ListResult
     | DerivedAddress[]
     | SimpleResult;
 
-export type RedirectResult = KeyResult
+export type RedirectResult = ExportResult
+    | KeyResult
```

Comment:
Would it be possible to have all combined results in separate lines, instead of the first one not? Just for code cleanliness :stuck_out_tongue:

Reply:
Like so?

Reply:
Yes, lgtm.